source/features/infinite-scroll.tsx

```diff
@@ -52,11 +51,17 @@ function init(): void {
 			{footer}
 		</div>,
 	);
+
+	return [
+		loadMore.cancel,
+		selectorObserver.abort,
+		inView.disconnect,
+	];
```

Comment:
```suggestion
	return [loadMore, selectorObserver, inView];
```

Reply:
We do need to keep `loadMore.cancel` though (`loadMore` is just the de-bounced version)